packages/components/src/components/tooltip/tooltip.js

```diff
@@ -137,6 +137,12 @@ class Tooltip extends mixin(
         throw new Error('Cannot find the target tooltip.');
       }
 
+      // Set up focus to be moved to the content on tooltips with no interactive elements
+      const content = tooltip.querySelector(this.options.selectorContent);
+      if (content.querySelector(settings.selectorTabbable) === null) {
+        content.setAttribute('data-floating-menu-primary-focus', '');
```

Comment:
Would we introduce `selectorFocusable`, which basically is a copy of [`selectorTabbable`](https://github.com/carbon-design-system/carbon/blob/v10.7.0/packages/components/src/globals/js/settings.js#L30-L35) with `:not([tabindex='-1'])` removed? And update [`floating-menu.js` code](https://github.com/carbon-design-system/carbon/blob/v10.7.0/packages/components/src/components/floating-menu/floating-menu.js#L325-L335) to focus on first (programmatically) focusable element if attempts to focus on `[data-floating-menu-primary-focus]` and sequentially focusable element fails?